tests/manager/manager-ctor-auto_encryption-error-001.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): incomplete auto encryption options
```

Comment:
This would also benefit from a `--SKIPIF--` that calls `skip_if_not_libmongocrypt()`.

I'll defer to you if you'd like to create a separate test for the `phongo_manager_set_auto_encryption_opts()` exception message when FLE is not available. If so, that would warrant creating another function: `skip_if_libmongocrypt()`.